Lib/sqlite3/dbapi2.py

```diff
@@ -45,7 +48,8 @@ def TimeFromTicks(ticks):
 def TimestampFromTicks(ticks):
     return Timestamp(*time.localtime(ticks)[:6])
 
-version_info = tuple([int(x) for x in version.split(".")])
+_deprecated_version_info = tuple(
+    [int(x) for x in _deprecated_version.split(".")])
```

Comment:
Tiny speedup and fits on one line (if we want to keep the comprehension version, we should drop the inner listcomp.)
```suggestion
_deprecated_version_info = tuple(map(int, _deprecated_version.split(".")))
```